service/src/test/java/tds/config/services/impl/ConfigServiceUnitTests.java

```diff
@@ -32,31 +31,51 @@ public void Setup() {
         configService = new ConfigServiceImpl(mockConfigRepository);
     }
 
+    @After
+    public void Teardown() {
+        mockConfigRepository = null;
+        configService = null;
```

Comment:
i don't think there is any harm in doing this but since those objects don't hold any external resources or anything you don't need to.  they will just get garbage collected